tests/integration_test/core_http_system_test.c

```diff
@@ -258,14 +308,55 @@ static int32_t transportSendStub( NetworkContext_t * pNetworkContext,
 
 /*-----------------------------------------------------------*/
 
+
+static int32_t generateRandomNumber()
+{
+    const uint32_t multiplier = 0x015a4e35UL, increment = 1UL;
+
+    nextRand = ( multiplier * nextRand ) + increment;
+    return( ( int32_t ) ( nextRand >> 16UL ) & 0x7fffUL );
+}
+
+static void seedRandomNumberGenerator()
```

Comment:
Where is this seeding done in this test?

Reply:
Good catch, have added the call in `TEST_SETUP`